### PR TITLE
Views

### PR DESCRIPTION
**Pull request by gaks1**
- Add the index and show views for the users and posts.
-  In the users index - list out all users
- In the users show - list a user and 3 recent posts
- In the posts index - list a user his posts and 5 recent commetns
- In the posts show -list a post and all its comments

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find_by_id(params[:user_id])
+    @posts = @user.posts
+  end
```
Comment (DuaneDave) [inline app/controllers/posts_controller.rb:5]: Good job having all the setups with your controllers, however when I try to create a post, it doesn't work and this makes it impossible to know if your controller actions are working, consider adding this `  t.integer :author_id` to your post migration file in order for posts to be created successfully.


[screen-capture (1).webm](https://github.com/gaks1/ruby-blog-app/assets/84930355/1767c04f-faf4-40c4-8757-454f9ec7c168)

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @gaks1 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good controller set up
✔️ Good views set up


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (gaks1) [reply app/controllers/posts_controller.rb:5]: I believe it is because of the post_counter it seems it needs to be initialized as zero

Comment (gaks1) [reply app/controllers/posts_controller.rb:5]: @DuaneDave

Comment (gaks1) [reply app/controllers/posts_controller.rb:5]: the comments_counter and likes_counter also need to be zero like    first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my 1 post', comments_counter: 0, likes_counter: 0)

Comment (gaks1) [reply app/controllers/posts_controller.rb:5]: @DuaneDave

Comment (gaks1) [reply app/controllers/posts_controller.rb:5]: the comments_counter and likes_counter also need to be zero like first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my 1 post', comments_counter: 0, likes_counter: 0) @DuaneDave

**Review by uwadonat — APPROVED** [review]
## Hi @gaks1,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉
### Highlights
- Correct workflow was used ✔️
- Good project structure ✔️
- Professional readme file ✔️
- Well-organized and clean project ✔
- Good commit history ✔️
thanks-awesome


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**